modules/publicize/publicize.php

```diff
@@ -1038,6 +1038,12 @@ public function update_published_message( $messages ) {
 		if ( ! $this->post_type_is_publicizeable( $post_type ) ) {
 			return $messages;
 		}
+
+		// Bail early if the post is private.
+		if ( 'private' === $post->post_status ) {
```

Comment:
It's probably slightly more correct to look at the return value of `get_post_status_object()`:

```php
$post_status_object = get_post_status_object( $post->post_status );
if ( $post_status_object->private ) {
  return $messages;
}
```

Or maybe
```php
if ( ! $post_status_object->public ) {
  return $messages;
}
```

I'm not sure.

That said, none of the other Publicize code does that :) Instead, we only Publicize posts that have `post_status=publish`, which suggests to me that we shouldn't bother with the post_status_object at all, but that we should change the conditional to:

```php
if ( 'publish' !== $post->post_status ) {
  return $messages;
}
```

What do you think?

Reply:
That's true. Updated in 6d1cefa4e4d6efdb02daf630867b02763088835b